### PR TITLE
Added a Disk Size parameter for Google Compute Engine.

### PR DESCRIPTION
**Pull request by saturnism**
Signed-off-by: Ray Tsang <saturnism@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +129,7 @@ Options:
 
  - `--google-zone`: The zone to launch the instance.  Default: `us-central1-a`
  - `--google-machine-type`: The type of instance.  Default: `f1-micro`
+ - `--google-disk-size`: The disk size of the instance (in GiB).  Default: `10`
```
Comment (ehazlett) [inline README.md]: could you change this to `GB` to keep consistency with the rest of the docs?

**Comment by ehazlett** [comment]
Tested on GCE.  Works as expected.

**Comment by saturnism** [comment]
Changed GiB to GB.  Cheers!

**Comment by ehazlett** [comment]
LGTM

**Pull request merged**